src/snapshot.ts

```diff
@@ -0,0 +1,346 @@
+import {
+  bufferToFile,
+  fileToBuffer,
+  mySessionItems,
+  mySessionMeat,
+  print,
+  sellPrice,
+  toInt,
+  toItem,
+} from "kolmafia";
+import { $item, $items, getFoldGroup, getSaleValue, property, set, sumNumbers } from "libram";
+import { fmt, HIGHLIGHT, resetDailyPreference } from "./lib";
+
+/**
+ * Return a mapping of the session items, mapping foldable items to a single of their forms
+ * @returns the item session results, with foldables mapped to a single of their folding forms
+ */
+function mySessionItemsWrapper(): Map<Item, number> {
+  const foldableMapping = (item: Item): [Item, Item][] =>
```

Comment:
I feel like I'd rather this be a `[Item, Item[]]` or, better yet a `{ representative: Item, group: Item[] }`, but that's neither here nor there

Reply:
It needs to be [Item, Item][] because it is beating a list of one to one , not one too many mappings